azkaban-common/src/main/java/azkaban/jobExecutor/ProcessJob.java

```diff
@@ -50,6 +52,13 @@ public void run() throws Exception {
       handleError("Bad property definition! " + e.getMessage(), e);
     }
 
+    Pair<Long, Long> memPair = getProcMemoryRequirement();
+    boolean memGranted = SystemMemoryInfo.requestMemory(memPair.getFirst(), memPair.getSecond());
```

Comment:
isMemoryAvailable (or something similar) might be a better variable name for the boolean, because we are not actually asking for memory like how malloc is doing it.
Otherwise, at least name it in the boolean variable name convention, e.g. isMemGranted